pom.xml

```diff
@@ -549,7 +549,7 @@
                 <groupId>junit</groupId>
                 <artifactId>junit</artifactId>
                 <version>${junit.version}</version>
-                <scope>test</scope>
+                <scope>compile</scope>
```

Comment:
This has very big consequences. We cannot change this, it must stay test scope.

This means the junit testing library is now included in every single module at compile time, not just for tests.